Release4.md

```diff
@@ -1,36 +1,123 @@
+
 ## Component Versions and User-visible improvements (highlights)
 
+* [OpenStack Zed release](https://releases.openstack.org/zed/highlights.html)
+* Ceph Quincy is available, the default release of Ceph is still Pacific.
+* The base infrastructure is provided by
+  [OSISM 5.0.0](https://release.osism.tech/notes/5.0.0.html)
+  which in turn builds on top of kolla and kolla-ansible.
+* With [Cloud-in-a-Box](https://github.com/osism/cloud-in-a-box) there is an easy way to get SCS up and running on a single hardware node as a test environment. There are two blog posts ([part 1](https://scs.community/2023/03/15/ciab/) and [part2](https://scs.community/2023/03/15/ciab-2/)) covering it.
+* For new deployments of the IaaS reference implementation Ubuntu 22.04 is recommended while existing installations can be upgraded to R4 while staying on Ubuntu 20.04. With Release 5, upgrading to Ubuntu 22.04 will be required.
+* With [osism/node-image](https://github.com/osism/node-image) an iso image for much easier bootstrapping of new OSISM environments is available now
+* The software for our [Kubernetes Cluster-API reference implementation](https://github.com/SovereignCloudStack/k8s-cluster-api-provider) has been updated and highlights are covered in own [release notes](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/main/Release-Notes-R4.md).<!--FIXME: We should link to docs.scs.community here.-->
+
+
 ## New Features (Highlights)
 
 ### Operator focused improvements
 
+* The [Openstack Image Manager](https://github.com/osism/openstack-image-manager) has seen many improvements and is the reference command to assure the images available comply with the [SCS Image Standard]( ) // FIXME LINK
+* For Ceph, special playbooks were added to validate the deployment status of the OSD, MON and MGR services in OSISM. The commands for use are `osism validate ceph-osds`, `osism validate ceph-mons`, and `osism validate ceph-mgrs`.
+* The [testbed](https://github.com/osism/testbed) uses per default a proxy for container pulling. This will allow for airgapped installations out of the box. Please note: a full airgap support (with local mirrors, etc.) will follow in a future release.
+* The efforts to create a well-maintained status page with well-defined interfaces resulted in an OpenAPI specification (within its own [repository](https://github.com/SovereignCloudStack/status-page-openapi)) which is intended to be implementable by multiple implementations. An actual reference implementation of this API is supposed to follow.
+* The dashboard of the [OpenStack Health Monitor](https://github.com/SovereignCloudStack/openstack-health-monitor/) is in use by the SCS operators and has proven helpful a number of times in detecting and addressing issues. That said, it only received a few fixes and minor enhancements, as we plan to replace it with a more generic and more maintainable solution soon.
```

Comment:
```suggestion
* The dashboard of the [OpenStack Health Monitor](https://github.com/SovereignCloudStack/openstack-health-monitor/) is in use by the SCS operators and has proven helpful a number of times in detecting and addressing issues.
```

Same as previous comments. Furthermore I think, it is not needed to downplay work by phrasing "it only received a few fixed and minor enhancements". That of course is subject to personal taste, but I'd opt for removing that sentence.

Reply:
Directing attention to a successor instead of to the old code was the intention here.
We could push that to a blog post instead though, indeed.

Reply:
Lets leave it as is then.